jobs/configure-eirini/templates/eirini-daemonset.yaml.erb

```diff
@@ -63,18 +46,15 @@ spec:
         - name: varlibdockercontainers
           mountPath: /var/lib/docker/containers
           readOnly: true
-        - name: varvcapstore
-          mountPath: /var/vcap/store
-          readOnly: true
-        - name: varvcapdata
-          mountPath: /var/vcap/data
-          readOnly: true
```

Comment:
This mount point wasn't need for us -- do you all need access to /var/vcap/data?

We also scoped down the mount for /var/vcap/store to be just the directory needed to pull out docker logs. We're using docker as the container runtime. Not sure how logging is handled with containerd or other runtimes.

Reply:
According to [this](https://github.com/cloudfoundry-community/eirini-bosh-release/blob/master/jobs/configure-eirini/templates/fluentd-configmap.yaml.erb#L14) and [this](https://github.com/cloudfoundry-community/eirini-bosh-release/blob/b55a8b50c9acb1208190c9121e57b1b10dc9b3b5/scripts/post-deploy-configure-k8s.sh#L93) what we need to mount is `/var/log` to allow fluentd to read the logs. I'm not sure about the other volumes. The fastest way would be to try to delete them one by one and see what breaks I guess.